Zend/zend_vm_def.h

```diff
@@ -9556,6 +9556,151 @@ ZEND_VM_HANDLER(202, ZEND_CALLABLE_CONVERT, UNUSED, UNUSED)
 	ZEND_VM_NEXT_OPCODE();
 }
 
+ZEND_VM_HANDLER(208, ZEND_JMP_FRAMELESS, CONST, JMP_ADDR, NUM|CACHE_SLOT)
+{
+	USE_OPLINE
+	uintptr_t result = (uintptr_t)CACHED_PTR(opline->extended_value);
+	switch (result) {
+		case 0: {
+			zval *func_name = (zval *)RT_CONSTANT(opline, opline->op1);
+			zval *func = zend_hash_find_known_hash(EG(function_table), Z_STR_P(func_name));
+			result = (func == NULL) + 1;
```

Comment:
I didn't get this. Why all functions not-found in ``EG(function_table)`` are frameless?

Reply:
Got it.

May be it makes sense to unite this instruction with a specialized version of ``INIT_FCALL_BY_NAME``. (don't do it right now. JIT part may be too complex).
Also we may make ``JMP_FRAMELESS`` and the following ``INIT_FCALL_BY_NAME`` to reuse the same cache slot. This shouldn't be difficult, and the resulting code should be both faster and more clean (I may be wrong).

Otherwise some comments at ``JMP_FRAMELESS`` should be added (like you explained this in the comment above).

Reply:
Done. I also added an enum for the magic constants, which hopefully makes this more obvious.